code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -884,3 +884,68 @@
 
 /datum/reagent/toxin/mimesbane/on_mob_delete(mob/living/L)
 	L.remove_trait(TRAIT_EMOTEMUTE, id)
+
+/datum/reagent/toxin/bonehurtingjuice //oof ouch
+	name = "Bone Hurting Juice"
+	id = "bonehurtingjuice"
+	description = "A strange substance that looks a lot like water. Drinking it is oddly tempting. Oof ouch."
+	color = "#AAAAAA77"
+	toxpwr = 0
+	taste_description = "bone hurting"
+	overdose_threshold = 50
+	metabolization_rate = 2.0
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_add(mob/living/carbon/M)
+	M.say("oof ouch my bones")
+
+/datum/reagent/toxin/bonehurtingjuice/on_mob_life(mob/living/carbon/M)
+	M.adjustStaminaLoss(8.0, 0)
+	if(prob(20))
+		switch(pick(1, 2, 3))
+			if(1)
+				var/list/possibleSays = list("oof", "ouch", "my bones", "oof ouch", "oof ouch my bones")
+				M.say(pick(possibleSays))
+			if(2)
+				var/list/possibleMes = list("oofs softly", "looks like their bones hurt", "grimaces, as though their bones hurt")
+				M.say("*custom " + pick(possibleMes))
+			if(3)
+				to_chat(M, "<span class='warning'>Your bones hurt!</span>")
+	. = ..()
+
+/datum/reagent/toxin/bonehurtingjuice/overdose_process(mob/living/M)
+	var/obj/item/bodypart/l_arm = M.get_bodypart(BODY_ZONE_L_ARM)
+	var/obj/item/bodypart/r_arm = M.get_bodypart(BODY_ZONE_R_ARM)
+	var/obj/item/bodypart/l_leg = M.get_bodypart(BODY_ZONE_L_LEG)
+	var/obj/item/bodypart/r_leg = M.get_bodypart(BODY_ZONE_R_LEG)
+	. = ..()
+	if(prob(4) && iscarbon(M) && (r_arm || l_arm || l_leg || r_leg)) //big oof
+		var/list/possibleSounds = list('sound/misc/desceration-01.ogg','sound/misc/desceration-02.ogg','sound/misc/desceration-01.ogg')
+		var/extMessage = "<span class='warning'>[M]'s bones hurt too much!!</span>"  //I couldn't figure out how to define a new function so let's not repeat the same strings 4 times.
+		var/playerMessage = "<span class='danger'>Your bones hurt too much!!</span>"
+		var/ouchie = "OOF!!"
+		var/stamDamage = 200
```

Comment:
None of these 4 need to be vars

Reply:
stam damage will stay as a var to make changing the limb damage possible with just one variable.  The other three were only vars because of really bad coding so they've been dropped.